wallet-api: Get rid of .TH modules

j-mueller opened the pull request:
Part of #1010

michaelpj commented:
Looks like we're having a line endings war. Can you check your settings? I think it should be on "check in Unix line endings", or at least that's what we've been doing in the past.

j-mueller commented:
Hm I thought I had fixed that.

I changed the line endings in `plutus-emulator/test/Spec.hs` to LFs, all the other files in this PR seemed to be OK.

michaelpj commented:
:+1:
